cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/validator/AzureIDBrokerObjectStorageValidator.java

```diff
@@ -124,6 +124,9 @@ private void validateLog(AzureClient client, PagedList<RoleAssignmentInner> role
             Set<String> mappedIdentityIds = new HashSet<>();
             mappedIdentityIds.addAll(accountMappings.getUserMappings().values());
             mappedIdentityIds.addAll(accountMappings.getGroupMappings().values());
+            mappedIdentityIds = mappedIdentityIds.stream()
+                    .map(id -> id.replaceFirst("(?i)/resourceGroups/", "/resourcegroups/"))
```

Comment:
What does the (?i) mean?

Reply:
ignore case, so "ResourceGroups" or "RESOURCEGROUPS" will be also converted to lowercase